Controllers/watch-later.js

```diff
@@ -0,0 +1,36 @@
+const { extend } = require("lodash");
+const postWatchLater = async (req, res) => {
+  try {
+    let { user } = req;
+    const watchlater = req.body;
+    user.watch_later.push(watchlater);
```

Comment:
Is the logic to handle redundancy handled on frontend?

Reply:
Yes, it is made to handle redundancy on frontend